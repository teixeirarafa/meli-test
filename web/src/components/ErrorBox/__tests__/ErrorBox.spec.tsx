import React from "react";
import { render } from "@testing-library/react";
import ErrorBox from "..";

describe("ErrorBox Component", () => {
  it("should render ErrorBox", () => {
    const app = render(<ErrorBox />);

    expect(app).toMatchSnapshot();
  });
});
