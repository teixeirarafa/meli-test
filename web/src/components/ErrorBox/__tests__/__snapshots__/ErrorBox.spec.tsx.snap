// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBox Component should render ErrorBox 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container"
      >
        <section
          class="error box padding--16"
        >
          <picture
            class="error__icon margin-right--16"
          >
            <img
              alt="Icon error"
              src="error.svg"
            />
          </picture>
          <div
            class="error__message"
          >
            <h2
              class="error__message-title"
            >
              Erro!
            </h2>
            <p
              class="error__message-description"
            />
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container"
    >
      <section
        class="error box padding--16"
      >
        <picture
          class="error__icon margin-right--16"
        >
          <img
            alt="Icon error"
            src="error.svg"
          />
        </picture>
        <div
          class="error__message"
        >
          <h2
            class="error__message-title"
          >
            Erro!
          </h2>
          <p
            class="error__message-description"
          />
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
